### PR TITLE
Add GraphQL service example

### PR DESCRIPTION
**Pull request by sameerajayasoma**
This PR contains a GraphQL sample that showcases the GraphQL support in Ballerina.

### CODE REVIEW

File: graphql_bookstore_service/main.bal [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        return self.bookData.year;
+    }
+
+    resource function get author() returns string {
+        return self.bookData.author;
+    }
+
+    resource function get reviews() returns Review|error {
+        string isbn = self.bookData.isbn;
+        GoogleBook googleBook = check bookEp->/books/v1/volumes.get(q=string `isbn:${isbn}`);
```
Comment (anupama-pathirage) [inline graphql_bookstore_service/main.bal]:
```suggestion
        GoogleBook googleBook = check bookEp->/books/v1/volumes.get(q = string `isbn:${isbn}`);
```

File: graphql_bookstore_service/README.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
+# GraphQL Bookstore Service
+This sample exposes a GraphQL service.
+
+## Use case
+In this use case, let’s see how we can implement a GraphQL server using the Ballerina to expose data in memory and data retrieved via another API call.
+
+The information sources for the above operations are as follows.
+* Title, published year, ISBN number, author  - Retrieved from the datastore backed by an in-memory table.
+* Average rating and rating count - Retrieved from Google  Books API filtered using ISBN number of the book.
```
Comment (anupama-pathirage) [inline graphql_bookstore_service/README.md]:
```suggestion
* Average rating and rating count - Retrieved from Google  Books API filtered using the ISBN number of the book.  
```

File: graphql_bookstore_service/README.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+In this use case, let’s see how we can implement a GraphQL server using the Ballerina to expose data in memory and data retrieved via another API call.
+
+The information sources for the above operations are as follows.
+* Title, published year, ISBN number, author  - Retrieved from the datastore backed by an in-memory table.
+* Average rating and rating count - Retrieved from Google  Books API filtered using ISBN number of the book.  
+E.g.: https://www.googleapis.com/books/v1/volumes?q=isbn:9781101042472
+
+# Run the code
+
+Execute `bal run` command  within the `graphql_bookstore_service` package directory.
```
Comment (anupama-pathirage) [inline graphql_bookstore_service/README.md]:
```suggestion
Execute `bal run` command within the `graphql_bookstore_service` package directory.
```

File: graphql_bookstore_service/README.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+ }
+}
+```
+
+CURL Command to send the same request:
+```
+curl -X POST -H "Content-type: application/json" -d '{ "query": "mutation {addBook(isbn: \"9781683836223\", title: \"Harry Potter\", author: \"J. K. Rowling\", year: 2007){isbn}}" }' 'http://localhost:9090/graphql'
+```
+
+You can find the original sample here: [https://github.com/anupama-pathirage/ballerina-scenarios/tree/main/ballerina-graphql-with-multiple-datasources](https://github.com/anupama-pathirage/ballerina-scenarios/tree/main/ballerina-graphql-with-multiple-datasources)
```
Comment (anupama-pathirage) [inline graphql_bookstore_service/README.md]: IMO, this is not needed as it is like we are referring to personal repo from the official guides repo :)

**Review by anupama-pathirage — DISMISSED** [review]

File: graphql_bookstore_service/README.md (hunk 4 above)
Comment (sameerajayasoma) [reply graphql_bookstore_service/README.md]: Updated the file :)

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**